index.html

```diff
@@ -2758,81 +2759,95 @@ <h2 class="icon-title">
             Icons with "maskable" purpose
           </h2>
           <div class="icons">
-            <div>
+            <figure>
               <img src="images/icon-plain.svg" alt=
               "An icon over a checkerboard background">
-              <div class="icon-title">
-                Image
-              </div>
-              <div>
-                The base image with transparent background
-              </div>
-            </div>
-            <div>
+              <figcaption>
+                <div class="icon-title">
+                  Image
+                </div>
+                <div>
+                  The base image with transparent background
+                </div>
+              </figcaption>
+            </figure>
+            <figure>
               <img src="images/icon-safe-zone.svg" alt=
               "An icon in a purple circle (40% of the size) over a yellow background">
-              <div class="icon-title">
-                Safe zone
-              </div>
-              <div>
-                Circle with radius 2/5 (40%) of the icon size
-              </div>
-            </div>
+              <figcaption>
+                <div class="icon-title">
+                  Safe zone
+                </div>
+                <div>
+                  Circle with radius 2/5 (40%) of the icon size
+                </div>
+              </figcaption>
+            </figure>
           </div>
           <h2 class="icon-title">
             Mask examples
           </h2>
           <div class="icons">
-            <div>
+            <figure>
               <img src="images/icon-mask-android-roundsquare.svg" alt=
               "An icon inside a rounded yellow square on a purple background">
-              <div class="icon-title">
-                Rounded square
-              </div>
-              <div>
-                Android
-              </div>
-            </div>
-            <div>
+              <figcaption>
+                <div class="icon-title">
+                  Rounded square
+                </div>
+                <div>
+                  Android
+                </div>
+              </figcaption>
+            </figure>
+            <figure>
               <img src="images/icon-mask-android-squircle.svg" alt=
               "An icon inside an extremely rounded yellow square on a purple background">
-              <div class="icon-title">
-                Squircle
-              </div>
-              <div>
-                Android
-              </div>
-            </div>
-            <div>
+              <figcaption>
+                <div class="icon-title">
```

Comment:
The validator seems to want these to be `span`s, which is causing Travis to reject the PR.

Reply:
Fixed!